.travis.yml

```diff
@@ -1,22 +1,25 @@
 language: php
 php:
-  - 7.0
-  - 5.6
-  - 5.5
-  - hhvm
```

Comment:
I think it will work on HHVM. It is enabled by default so no extra config is needed.